downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc

```diff
@@ -1,4 +1,4 @@
-// this info is duplicated here: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/installing_and_upgrading_private_automation_hub/installing_online_or_offline#doc-wrapper
+// this info is duplicated here: {BaseURL}/red_hat_ansible_automation_platform/1.2/html/installing_and_upgrading_private_automation_hub/installing_online_or_offline#doc-wrapper
```

Comment:
Is this the link that's actually included? To 1.2?

Reply:
No, it looks like a comment. It's an ancient link.